stablehlo/dialect/StablehloOps.cpp

```diff
@@ -4499,10 +4502,14 @@ struct HLOInlinerInterface : public DialectInlinerInterface {
   }
 };
 
-struct HLOBoundedDialectInterface : public hlo::BoundedDialectInterface {
```

Comment:
(no action required)
Just did a quick search - looks like these were the only instances of the `HLO` in all caps naming convention, good to clean this up. Similarly for MHLO codebase these two structs are the only using this variable casing.